frontend/user/vue-components/ApplicationListView.vue

```diff
@@ -1,15 +1,5 @@
 <template>
   <section class="sidebar">
-  <!-- Error dialog -->
```

Comment:
Now it's in ApplicationLabel.vue, and I had some test for it in the corresponding test file